torch/csrc/jit/codegen/cuda/fusion_segmenter.h

```diff
@@ -312,6 +312,17 @@ class TORCH_CUDA_CU_API SegmentedFusion {
   //! API for adding edges
   SegmentedEdge* newEdge(SegmentedGroup* from, SegmentedGroup* to, Val* val);
 
+  //! Returns the set of potential intermediate tensors that
+  //!  will be cast to fp16 when written to global mem.
+  //!  These are not actual intermediate tensors,
+  //!  just the ones that will need to cast to fp16 if
+  //!  they end up being an intermediate tensor between
+  //!  segmented groups.
+  const auto& getForceToFP16Set() {
```

Comment:
nitpick: do we actually need this function. Just realized it wasn't actually used.